src/config_cache.c

```diff
@@ -86,7 +86,8 @@ int git_config__cvar(int *out, git_config *config, git_cvar_cached cvar)
 	struct map_data *data = &_cvar_maps[(int)cvar];
 	git_config_entry *entry;
 
-	git_config__lookup_entry(&entry, config, data->cvar_name, false);
+	if ((error = git_config__lookup_entry(&entry, config, data->cvar_name, false)) < 0)
+		return error;
```

Comment:
In fact it does not, see the commit message. `git_config__lookup_entry` with `no_errors == false` calls `get_entry` with `GET_NO_MISSING`. So we shouldn't get an error for missing entries.